client/my-sites/plugins/plans/index.tsx

```diff
@@ -66,6 +68,36 @@ const Plans = ( { intervalType }: { intervalType: 'yearly' | 'monthly' } ) => {
 					isReskinned
 				/>
 			</div>
+
+			<div className="plugin-plans-step-hints">
+				<section>
```

Comment:
Can you use the [`PromoSection` component](https://github.com/Automattic/wp-calypso/blob/8f7dc753cfcf1614ad854243f9b2f4e975d22c88/client/my-sites/woocommerce/landing-page.tsx#L217) for these?

Reply:
- > Can you use the [PromoSection component](https://github.com/Automattic/wp-calypso/blob/8f7dc753cfcf1614ad854243f9b2f4e975d22c88/client/my-sites/woocommerce/landing-page.tsx#L217) for these?

	Great suggestion @lsl, thanks! [Updated it here](https://github.com/Automattic/wp-calypso/pull/68058/commits/c2656742a1acbe033b96483752b2d5b44fa2b315). 


- > for various icons / sizing / mobile handling reasons

	I recall my call yesterday with @vinimotaa, and one requirement would have less vertical space in both desktop and mobile versions. The [PromoSection component](https://github.com/Automattic/wp-calypso/blob/8f7dc753cfcf1614ad854243f9b2f4e975d22c88/client/my-sites/woocommerce/landing-page.tsx#L217) for default will use more vertical space and won't follow the design at Figma.

	Using the component out of the box, we only need a few lines of custom CSS (which would be great @vinimotaa ):
	
	<img width="1215" alt="image" src="https://user-images.githubusercontent.com/1044309/191496822-74550bb2-30dd-4d7c-b25a-7acedd90cca0.png">

	The downside is the mobile version uses more vertical space @vinimotaa:

	<img width="487" alt="image" src="https://user-images.githubusercontent.com/1044309/191496731-0e8cb67e-9851-4657-b7c3-9d134f125b92.png">



- > Updating the promo section component to match the new design is also an option but I advise against it.

	Can you please elaborate more on why? [On Woo implementation](https://github.com/Automattic/wp-calypso/blob/update/67977-new-step-hints/client/my-sites/woocommerce/style.scss#L89-L126) we also have extra CSS code on the promo section. So I'm curious to understand your vision.